.github/workflows/packaging/install_dependencies.sh

```diff
@@ -11,13 +11,13 @@ apt update \
     libgl1-mesa-dev \
     libglu1-mesa-dev
 
-## Build and install librealsense2 ##
-git clone https://github.com/IntelRealSense/librealsense.git
+## Build and install Intel RealSense SDK 2.0 v2.45.0 ##
+git clone https://github.com/IntelRealSense/librealsense.git -b v2.45.0
 
 PWD=$(pwd)
 
 cd librealsense
-cmake -Bbuild -H.
+cmake -Bbuild -DCMAKE_BUILD_TYPE=Release -DBUILD_EXAMPLES=false -H.
```

Comment:
please add `-DBUILD_GRAPHICAL_EXAMPLES=false`

Reply:
It's not required if you don't build the examples in the first place.

Reply:
@markusachtelik sorry actually I just saw they are not mutually exclusive. I will do a second PR.